CMakeLists.txt

```diff
@@ -218,13 +219,31 @@ check_symbol_exists("malloc_trim" "malloc.h" HAVE_MALLOC_TRIM)
 check_function_exists(mallinfo HAVE_MALLINFO)
 check_function_exists(mallinfo2 HAVE_MALLINFO2)
 
+check_symbol_exists("htonll" "sys/types.h;netinet/in.h;inttypes.h" HAVE_HTONLL)
+
 check_symbol_exists("eat_newline_glitch" "term.h" HAVE_EAT_NEWLINE_GLITCH)
 
+# Check if res_init requires libresolv
```

Comment:
There's one issue there with the FreeBSD build, see: https://github.com/weechat/weechat/actions/runs/15235521214/job/42848778119#step:3:2850

I'm looking how to fix, but if you have ideas, they are welcome (commit is already merged on main branch, so a separate PR is welcome if needed).